src/Symfony/Component/Cache/DataCollector/CacheDataCollector.php

```diff
@@ -0,0 +1,177 @@
+        $this->data = array('instances' => $empty, 'total' => $empty);
+        foreach ($this->instances as $name => $instance) {
+            $calls = $instance->getCalls();
+            foreach ($calls as $call) {
+                if (isset($call->result)) {
+                    $call->result = $this->cloneVar($call->result);
+                }
+                if (isset($call->argument)) {
+                    $call->argument = $this->cloneVar($call->argument);
+                }
+            }
+            $this->data['instances']['calls'][$name] = $calls;
+        }
+
+        $this->data['instances']['statistics'] = $this->calculateStatistics();
+        $this->data['total']['statistics'] = $this->calculateTotalStatistics();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'cache';
+    }
+
+    /**
+     * Method returns amount of logged Cache reads: "get" calls.
+     *
+     * @return array
+     */
+    public function getStatistics()
+    {
+        return $this->data['instances']['statistics'];
+    }
+
+    /**
+     * Method returns the statistic totals.
+     *
+     * @return array
+     */
+    public function getTotals()
+    {
+        return $this->data['total']['statistics'];
+    }
+
+    /**
+     * Method returns all logged Cache call objects.
+     *
+     * @return mixed
+     */
+    public function getCalls()
+    {
+        return $this->data['instances']['calls'];
+    }
+
+    /**
+     * @return array
+     */
+    private function calculateStatistics()
+    {
+        $statistics = array();
+        foreach ($this->data['instances']['calls'] as $name => $calls) {
+            $statistics[$name] = array(
+                'calls' => 0,
+                'time' => 0,
+                'reads' => 0,
+                'hits' => 0,
+                'misses' => 0,
+                'writes' => 0,
+                'deletes' => 0,
+            );
+            /** @var TraceableAdapterEvent $call */
+            foreach ($calls as $call) {
+                $statistics[$name]['calls'] += 1;
+                $statistics[$name]['time'] += $call->end - $call->start;
+                if ($call->name === 'getItem') {
+                    $statistics[$name]['reads'] += 1;
+                    if ($call->hits) {
+                        $statistics[$name]['hits'] += 1;
+                    } else {
+                        $statistics[$name]['misses'] += 1;
+                    }
+                } elseif ($call->name === 'getItems') {
+                    $count = $call->hits + $call->misses;
+                    $statistics[$name]['reads'] += $count;
+                    $statistics[$name]['hits'] += $call->hits;
+                    $statistics[$name]['misses'] += $count - $call->misses;
+                } elseif ($call->name === 'hasItem') {
+                    $statistics[$name]['reads'] += 1;
+                    if ($call->result->getRawData()[0][0] === false) {
```

Comment:
No better way for now, I may work on something better soon but this is the way to go for now.